doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst

```diff
@@ -130,40 +119,42 @@ The following are the prefixes used to indicate the cores:
-The host PC, equipped with a decoder tool, translates these addresses back into human-readable text using a JSON database generated during the build process.
+Assisted multicore logging uses dictionary-based logging to send messages without redundant strings to STM, and is based on the :ref:`zephyr:logging_guide_dictionary` feature of the logging API provided by Zephyr.
+For more information on assisted multicore logging, see :ref:`zephyr:logging_cs_stm`.
 
-When using logs, this method has the following advantages:
+Configuring logging
+-------------------
 
-* It reduces the size of the binary, as the strings used in the log messages are not stored in the binary itself.
-* It reduces the amount of data that needs to be sent to and processed by the application core, as the string formatting is offloaded to the host side.
+To show the logs for a given domain in the UART output, you must enable the STM for both the application core (as the ETR buffer processing proxy) and that domain's CPU.
+To do so, use the :ref:`zephyr:nordic-log-stm-dict` snippet when building the application for the related cores.
+For example, the following command uses the snippet when building for the application core::
 
-Configuration
--------------
+   west build -b nrf54h20dk/nrf54h20/cpuapp -S nordic-log-stm-dict
 
-.. to review
+All cores must use the same logging configuration.
 
-To enable the assisted multicore logging for a domain, set the :kconfig:option:`CONFIG_LOG_STM_DICT` Kconfig option for that domain's image to ``y``.
-
-After building your application, a dictionary database file named :file:`log_database.json` will be generated in the :file:`build/zephyr` directory.
+After building your application, a dictionary database file named :file:`log_database.json` will be generated in the build directories for each one of the cores the snippet was used on (:file:`build/<app_name>/zephyr/`, where ``<app_name>`` is the name of the application built for each specific core).
 This file is crucial for decoding the logs and is updated with every build.
 
 Reading the logs
 ----------------
 
-.. to review
-
 To read the dictionary-based STM log output, do the following:
 
 1. Set up the log capture.
 
    Use the ``nrfutil trace stm`` command to start capturing logs from the device, specifying the database configuration for each domain ID, as well as the serial port, the baud rate, and the output file name::
 
-      nrfutil trace stm --database-config 33:build/secdom/src/secdombuild/zephyr/log_dictionary.json,34:build/zephyr/log_dictionary.json --input-serialport /dev/ttyACM1 --baudrate 115200 --output-ascii out.txt
+      nrfutil trace stm --database-config <domain_id>:build/<app_name>/zephyr/log_dictionary.json --input-serialport <port> --baudrate 115200 --stdout ascii
+
+   This command contains the following parameters:
+
+   * ``<domain_id>`` is the ID of the domain which the application is running on.
```

Comment:
As domain_id should be in decimals, then to be consistent, at csv-table:: nRF54H20 log prefixes there should be also decimals - currently there are hexadecimals.

Reply:
@PavelVPV Could you check this?

Reply:
That was point 5.2 from my comment: https://github.com/nrfconnect/sdk-nrf/pull/17331#pullrequestreview-2324706941. It would be more friendlier if the table had ids in decimals, though I'm fine with how it is right now. Maybe there are other places where heximals are used. Up to you if you want to fix it or not.

Reply:
Could you provide the decimal values?